wiki/Mapping_and_Modding_Timeline/en.md

```diff
@@ -0,0 +1,453 @@
+    - This system mimicked the later stage of the [Mapping Assistance Team](/wiki/Modding/Mapping_Assistance_Team).
+        - Tier 1 nominators could bubble maps.
+        - Tier 2 nominators could bubble and qualify maps.
+    - Tiers were decided by a BN test designed around identifying overarching issues in beatmaps. Test results were terrible, revealing that the test's design was flawed.
+    - Contrary to its goals, this system lowered motivation for all tier 1 nominators and overworked the few motivated tier 2 nominators. Mappers had fewer BN resources and the total maps ranked during the tiers period diminished.
+    - No Beatmap Nominators were added to the 2nd tier for the 5 months this system was active, which hugely bottlenecked ranking speeds. <!-- https://osu.ppy.sh/community/forums/topics/578413 -->
+
+### May
+
+- **2017-05-22:** Beatmap Nominator Discord server was created for BN subdivisions.
+    - Subdivisions were designed as focus groups of randomly selected nominators from both tiers led by a member of the QAT.
+    - Most QAT management shifted to this server despite resistance from osu! higher-ups.
+- **2017-05-23:** Discord bot "Aiess" <!-- TODO: stub? --> was created by [Naxess](https://osu.ppy.sh/users/8129817).
+    - This provided a feed of mapping related events in osu! Discord servers, starting with nomination and disqualification feeds.
+
+### June
+
+- **2017-06-02:** Marathon beatmaps began to require 2 nominations instead of 3.
+    - They previously required 3 nominations because it was expected that marathons had more content to check than a normal mapset, but this was no longer true.
+    - The `flame` (![flame icon](/wiki/shared/icon/flame.gif)) icon discontinued use. <!-- https://osu.ppy.sh/community/forums/topics/631077?start=6050796 -->
+
+### September
+
+- **2017-09-10:** The "[QAT Upheaval](https://osu.ppy.sh/community/forums/topics/635507)" was implemented. This was a series of changes in response to dissatisfaction from the mapping community under leadership by [osu!team](/wiki/People/The_Team) higher-ups. Following this change, the QAT began self-leadership without higher-up intervention.
+    - BN tiers were replaced with [Probationary Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators#probationary-beatmap-nominators). Two probationary BN members could not nominate the same beatmap and users could not be in Probation for more than two months at a time.
+    - Beatmap Nominators were given purple titles on the forums.
+    - [Okoratu](https://osu.ppy.sh/users/1623405) and [JBHyperion](https://osu.ppy.sh/users/4879508) were appointed as QAT leaders <!-- TODO: stub for QAT Leaders --> by a vote of BN/QAT members. Their responsibility was to work with the osu!team to maintain the mapping and modding communities. 
+    - *Elite Nominator* profile badges and forum titles were rewarded to exceptional Beatmap Nominators.
```

Comment:
uhhhh, pretty sure this was at the end of 2019, I never heard of anything similar happening in 2017.